src/request/export/ExportWords.js

```diff
@@ -148,6 +160,21 @@ class ExportWords extends Nimiq.Observable {
         this._validateWords.setWords(words);
 
         window.location.hash = ExportWords.Pages.SHOW_WORDS;
+
+        /*
+         * scrollListener is set up here as otherwise the element is display: none;
+         * and therefore would have offsetHeight === 0
+         */
+        const scrollTopThreshold = /** @type {HTMLElement} */(this.$recoveryWords.children.item(0)).offsetHeight
+            + parseInt(window.getComputedStyle(this.$recoveryWords, null).getPropertyValue('padding-top'), 10)
+            + parseInt(window.getComputedStyle(this.$recoveryWords, null).getPropertyValue('padding-bottom'), 10) / 2
+            - this.$recoveryWords.offsetHeight;
+        this.$recoveryWords.onscroll = () => {
+            if (this.$recoveryWords.scrollTop > scrollTopThreshold) {
```

Comment:
No need for complicated calulations.
Use `scollHeight` insted, e.g.: `this.$recoveryWords.scrollTop >= this.$recoveryWords.offsetHeight - x`